impl_test.go

```diff
@@ -118,7 +118,7 @@ func TestFuncs(t *testing.T) {
 				},
 				{
 					Name:   "WriteHeader",
-					Params: []Param{{Type: "int"}},
+					Params: []Param{{Type: "int", Name: "statusCode"}},
```

Comment:
I had to add this to make the tests pass in Go 1.10. Looks like the interface changed somewhere along the line.